R/GRanges_helpers.R

```diff
@@ -102,22 +103,47 @@ strandPerGroup <- function(grl, keep.names = T){
   }
 }
 
-#' get first exon per granges group
+#' helper function to get a logical list of True/False, if GRangesList group have
+#' + strand = T, if - strand = F
+#' Also checks for * strands, so a good check for bugs
+#' @param grl a GRangesList or GRanges object
+strandBool <- function(grl){
```

Comment:
The idea of * was that if user wants something to be counted towards two strands eg. RNA-Seq reads he can do that with *. With this you kind of restrict users to either fwd or rve strands. Whether to change it depends what functions use your helper.